nrf_dm/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(CONFIG_SOC_NRF52840)
+	set(NRF_DM_PATH ${CMAKE_CURRENT_SOURCE_DIR}/lib/pca10056/hard-float)
+elseif(CONFIG_SOC_NRF52833)
+	set(NRF_DM_PATH ${CMAKE_CURRENT_SOURCE_DIR}/lib/pca10100/hard-float)
+else()
+	message(FATAL_ERROR "Unsupported SoC.")
+endif()
```

Comment:
You should consider unifying the path names to other libraries. This will also make it more future proof.
See the usage of `nrfxlib_calculate_lib_path` for other libs.

Reply:
Good point @rugeGerritsen. If you decide not to fully align the _nrf_dm_ paths at least please consider getting rid of the "_pca10xxx_" naming as Zephyr and NCS don't use these codes. E.g. Zephyr defines build target `nrf52dk_nrf52832` where `nrf52dk` refers to the name of the DK.